### ISSUE TITLE
Error installing CVC4 on Ubuntu 20.04 (Python 3.8.10) 

### ISSUE THREAD

**Issue by danyalette** [pysmt/pysmt#707]
When using PySMT v0.9.0, the command `pysmt-install --cvc4` fails with the following error message: 

``` 
File "/home/<user>/.local/bin/pysmt-install", line 8, in <module>
    sys.exit(main())
  File "/home/<user>/.local/lib/python3.8/site-packages/pysmt/cmd/install.py", line 259, in main
    installer.install(force_redo=options.force_redo)
  File "/home/<user>/.local/lib/python3.8/site-packages/pysmt/cmd/installers/base.py", line 149, in install
    self.compile()
  File "/home/<user>/.local/lib/python3.8/site-packages/pysmt/cmd/installers/cvc4.py", line 49, in compile
    SolverInstaller.run("bash %s" % os.path.join("contrib", "get-antlr-3.4"),
  File "/home/<user>/.local/lib/python3.8/site-packages/pysmt/cmd/installers/base.py", line 233, in run
    subprocess.check_call(program, env=environment,
  File "/usr/lib/python3.8/subprocess.py", line 359, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python3.8/subprocess.py", line 340, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/home/<user>/.smt_solvers/cvc4/CVC4-391ab9df6c3fd9a3771864900c1718534c1e4666'
```

As a temporary fix, I was able to successfully install cvc4 by doing the following: 

`cp -r cvc5-391ab9df6c3fd9a3771864900c1718534c1e4666 CVC4-391ab9df6c3fd9a3771864900c1718534c1e4666` 
since the former directory did in fact exist. 

This issue is resolved for me but I am leaving this here to save someone else some time :)

**Comment by merlinsun** [comment]
I also encountered this situation.
The successor of CVC4, cvc5, has been released. I would appreciate it if the developers could make a corresponding update to this.

**Comment by mikand** [comment]
@danyalette @merlinsun If you use the master version of pysmt (or the latest PyPi package: `pip install PySMT==0.9.1.dev139`) the installer should work fine. We are still using the final version of CVC4 (not cvc5).

We will update to cvc5 ASAP, but this will require some changes in the installer and some testing to see that if/how the API of the solver evolved.

**Comment by merlinsun** [comment]
Thanks, @mikand!
By the way, when parsing some scripts about regex or RoundingMode, pysmt seems to throw an error.
I think it would be really helpful if more comprehensive theories could be supported.

**Comment by marcogario** [comment]
The main issue described here is fixed by having made a release for 0.9.5 (as it was already fixed in 0.9.1dev139) https://github.com/pysmt/pysmt/issues/707#issuecomment-1055157565 

An upgrade to cvc5 is still desirable but non-blocking.
